src/credentials/loginManager.ts

```diff
@@ -17,6 +18,8 @@ import {
 } from './providers/credentials'
 import { CredentialsProviderManager } from './providers/credentialsProviderManager'
 
+const CONTEXT_KEY_ISCONNECTED = 'isConnected'
```

Comment:
Swap to something like `aws-toolkit.isConnected` (better yet, make a const in the constants file for `aws-toolkit`). Pretty sure contexts like this apply globally to VS Code extensions, so this could get tripped by another extension with an `isConnected` context.